testsuite/python/rdf.py

```diff
@@ -107,20 +107,23 @@ def test_mixed(self):
     def test_rdf_interface(self):
         # test setters and getters
         s = self.s
-        s.part.add(id=0, pos=[0, 0, 0], type=0)
-        s.part.add(id=1, pos=[0, 0, 0], type=1)
-        observable = espressomd.observables.RDF(ids1=s.part[:].id[0::2],
-                                                ids2=s.part[:].id[1::2],
+        s.part.add(pos=[0, 0, 0], type=0)
+        s.part.add(pos=[0, 0, 0], type=1)
+        s.part.add(pos=[0, 0, 0], type=0)
+        s.part.add(pos=[0, 0, 0], type=1)
+        pids1 = [s.part[:].id[0]]
+        pids2 = [s.part[:].id[1]]
+        observable = espressomd.observables.RDF(ids1=pids1, ids2=pids2,
```

Comment:
I dont get the change in this testcase.
Is there a reason to double the particle count and not use them in the observable?
Why dont we add them in one call and use the returned particle handles for the ids?

Reply:
Nice catch, that's a copy paste error, I meant to take a slice, not the first element. The idea is to initialize the object with `pids1=[0, 2]` and `pids2=[1, 3]` and later update the observable with `pids1=[0]` and `pids2=[1]` to check the script interface setter correctly updated the core object.